card/src/test/java/com/adyen/checkout/card/DefaultCardDelegateTest.kt

```diff
@@ -331,9 +336,7 @@ internal class DefaultCardDelegateTest {
 
                 with(requireNotNull(expectMostRecentItem())) {
                     assertTrue(isValid)
-                    assertEquals(FieldState(TEST_CARD_NUMBER, Validation.Valid), cardNumberState)
-                    assertEquals(FieldState(TEST_SECURITY_CODE, Validation.Valid), securityCodeState)
-                    assertEquals(FieldState(TEST_EXPIRY_DATE, Validation.Valid), expiryDateState)
+                    assertEquals(createOutputData(), this)
```

Comment:
@OscarSpruit ended up asserting the whole object when applicable (mostly in the valid scenarios)

Reply:
nice 🙌

Reply:
to make it even shorter 👀 
```Kotlin
assertEquals(createOutputData(), requireNotNull(expectMostRecentItem()))
```

Reply:
There's an `assertTrue(isValid)` above it though

Reply:
Ohhh sorry missed that one